pkg/kubelet/kubelet.go

```diff
@@ -1626,18 +1626,15 @@ func (kl *Kubelet) syncPod(pod *api.Pod, mirrorPod *api.Pod, runningPod kubecont
 		return err
 	}
 
-	// Starting phase:
-	ref, err := api.GetReference(pod)
```

Comment:
If we keep this line, `ref` could be used later in this function. That might be better than getting it every time we want to record an event.